templates/CTPUSHTEMPLATESCHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## CleverTap Push Templates SDK CHANGE LOG
 
+### Version 1.0.7 (November xx, 2022)
```

Comment:
this version is not updated in version.properties, this will result in aar for v 1.0.7 not getting created and rather aar for 1.0.6 be created  @william-ct

Reply:
@root-ansh-ct Please do the changes. @william-ct is on leave today